sdk/aqueduct/artifacts/generic_artifact.py

```diff
@@ -90,3 +95,23 @@ def describe(self) -> None:
         ]
         print(format_header_for_print(f"'{input_operator.name}' {self.type()} Artifact"))
         print(json.dumps(readable_dict, sort_keys=False, indent=4))
+
+    def system_metric(
+        self, metric_name: str, lazy: bool = False
+    ) -> numeric_artifact.NumericArtifact:
+        """Creates a system metric that represents the given system information from the previous @op that ran on the table.
+
+        Args:
+            metric_name:
+                name of system metric to retrieve for the table.
```

Comment:
Can we capitalize the first letter of each sentence? "Name of system metric..." I think that's what we do elsewhere in the codebase, and also what the google docstring style guide seems to indicate.

Essentially, let's treat docstrings as real english, with proper capitalization and periods.

Reply:
This comment applies for all the other docstrings in this PR.